src/migration-failure-worker/interface.ts

```diff
@@ -7,4 +7,6 @@ export interface Message {
   migrationFailed?: boolean;
   migrationId: number;
   source?: string;
+  service?: string;
```

Comment:
This file needs to be renamed into something Message.ts and reside optionally reside in a sub folder of interfaces. the name interface.ts is vague.